### ISSUE TITLE
Download

### ISSUE THREAD

**Issue by arttwart** [Langomatisch/hypixelskyblock#3]
Is there a way to download that now?

**Comment by TOLHavoc** [comment]
> Is there a way to download that now?

No, I think you would have to build it; a pre-built one would be nice though.
